exercises/practice/complex-numbers/complex_numbers.go

```diff
@@ -0,0 +1,43 @@
+package complex
+
+// Define the Number type here.
+
+func (n Number) Real() float64 {
+	panic("Please implement the Real function")
```

Comment:
```suggestion
	panic("Please implement the Real method")
```
Here we have methods and not really functions.

This feedback also applies to the other stubs.

Reply:
For me it is a receiver function, therefore I used function, will replace it :)